src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -191,6 +191,11 @@ public static class Sse2
         /// </summary>
         public static Vector128<ushort> Average(Vector128<ushort> left, Vector128<ushort> right) => Average(left, right);
 
+        /// <summary>
+        /// HELPER: MOVSD
+        /// </summary>
+        public static Vector128<double> CastDoubleToVector128Double(double value) => CastDoubleToVector128Double(value);
```

Comment:
Don't we need JIT implementations of these methods?  Today `Sse` and `Sse2` IsSupported returns `true`, so if someone tried calling these methods, they would get stack overflows, right?

Reply:
Yes, we do. I will implement them very quickly once the API gets approved.

Reply:
@eerhardt, we would need to provide an untested implementation.

Granted these are "trivial" enough that it shouldn't be an issue, but @CarolEidt would need to sign-off